apps/src/p5lab/P5LabView.jsx

```diff
@@ -77,6 +78,12 @@ class P5LabView extends React.Component {
     this.p5labTeacherUploadEnabled_ = experiments.isEnabled(
       experiments.P5LAB_TEACHER_UPLOAD
     );
+
+    // Indicate the context of the animation picker
+    let projectType = this.props.isBlockly
+      ? PICKER_TYPE.spritelab
+      : PICKER_TYPE.gamelab;
+    this.setState({projectType: projectType});
```

Comment:
this logic should likely go in a constructor so that it's clear that we're using a prop to set some initial state